astropy/io/votable/converters.py

```diff
@@ -579,9 +579,8 @@ def __init__(self, field, base, arraysize, config=None, pos=None):
         self.default[...] = self._base.default
 
     def parse(self, value, config=None, pos=None):
-        if config is None:
-            config = {}
-        if config['version_1_3_or_later'] and value == '':
+        if config is None: config = {}
```

Comment:
This change (putting the if statement on one line) is not needed - can you revert this back to 2 lines?